client/src/components/Footer/Footer.js

```diff
@@ -15,4 +15,6 @@ const Footer = () => (
   </div>
 );
 
+// export default Footer;
+
```

Comment:
Can you please remove this commented out line?

Reply:
Thank you!
 @jazhen  I actually wrote that piece of code trying to get rid of the `index.js` file and export default directly from the `footer.js` file and fix all imports where necessary. for some reason that broke the app even after fixing imports in other places.
![water](https://user-images.githubusercontent.com/64507182/193415495-8244137a-15f8-48ff-a6a4-9ed3ac0da25f.png)

I commented it out to get back to it later.
find out what the error is, see if I can fix then get rid of the `index.js` file

Reply:
but wait, when you say remove you meant get rid of it ? 

looks like I might be unclear what you meant, if it's to get rid of it. will do that
but if it's to uncomment it, that might cause some issues.

Reply:
Yup, just meant to get rid of it. Just wanted to make sure we don't have unused or in progress code committed if possible. Thank you!